internal/terminal/log.go

```diff
@@ -12,11 +12,16 @@ import (
 // LogLevel is the level of a terminal log
 type LogLevel string
 
-// set of supported log levels
```

Comment:
undo this change (it's failing the linter), you can put the unexported `linkMessage`/`commandMessage` in a separate `const` block, and maybe place that block closer to the constructor functions where these are used